batch/indexer_Position_Bond.py

```diff
@@ -172,7 +172,6 @@ def initial_sync(self):
             self.__set_idx_position_block_number(local_session, self.token_list, latest_block)
             local_session.commit()
         except Exception as e:
-            LOG.exception("An exception occurred during event synchronization")
```

Comment:
If exception occurs, batch will output warn/error log not in processor function but in main loop.